app.rb

```diff
@@ -147,34 +169,40 @@ def process_created_issue_comment(issue_comment_payload)
       @redis.hset(pr_key, current_commit_hash, payload_to_store.to_json)
 
       if plus_ones >= NEEDED_PLUS_ONES
-        # Set commit status to sucessful
-        @client.create_status(
-          pr_name,
-          current_commit_hash,
-          'success',
-          {
-            'description' => 'Commodus: Required plus ones (' + plus_ones.to_s + '/' + NEEDED_PLUS_ONES.to_s + ') has been reached!',
-            'context' => 'robinpowered/commodus'
-          }
-        )
+        status = 'success'
       else
-        @client.create_status(
-          pr_name,
-          current_commit_hash,
-          'pending',
-          {
-            'description' => 'Commodus: Required plus ones (' + plus_ones.to_s + '/' + NEEDED_PLUS_ONES.to_s + ') has yet to be reached.',
-            'context' => 'robinpowered/commodus'
-          }
-        )
+        status = 'pending'
       end
+
+      @client.create_status(
+        pr_name,
+        current_commit_hash,
+        status,
+        {
+          'description' => '(' + plus_ones.to_s + '/' + NEEDED_PLUS_ONES.to_s + ') required approvals.',
+          'context' => 'robinpowered/commodus'
+        }
+      ) 
     else
       return 404
     end
 
     return 200
   end
 
+  # Evaluates the PR review state
+  def evaluate_review_state(state)
+    net_pluses = 0
+
+    if state == "approved"
+      net_pluses = 1
+    elsif state == "changes_requested"
+      net_pluses = -1
+    end
+
+    return net_pluses
+  end
```

Comment:
Seems a bit overkill. **Anything** that is not `approved` or `changes_requested` should result in net 0.